ec2/ec2test/network_interfaces.go

```diff
@@ -404,6 +416,10 @@ func (srv *Server) createIFace(w http.ResponseWriter, req *http.Request, reqId s
 	return resp
 }
 
+func (srv *Server) dnsNameFromPrivateIP(privateIP string) string {
+	return fmt.Sprintf("ip-%s.ec2.internal", strings.Replace(privateIP, ".", "-", -1))
```

Comment:
The only reason for this change was to make both the local-live and live tests pass consistently (AWS does not populate DNSName sometimes).